src/nemo_run/core/execution/base.py

```diff
@@ -226,3 +227,49 @@ def package_configs(self, *cfgs: tuple[str, str]) -> list[str]:
         return filenames
 
     def cleanup(self, handle: str): ...
+
+
+def get_executor(
```

Comment:
Should we rename this to `import_executor`?

Reply:
Done in https://github.com/NVIDIA/NeMo-Run/pull/105/commits/b3146e64f12f11c0a9e4400efae8d0618f93eb23